run_segmentation.py

```diff
@@ -21,63 +27,58 @@ def run(self):
         os.environ["CUDA_VISIBLE_DEVICES"] = str(self.gpuid)
 
         if self.method == 'segmentation':
+            log.info(f"running segmentation worker on {self.gpuid}")
             segmentation(*self.inputs)
         elif self.method == 'instance_segmentation':
+            log.info(f"running instance segmentation worker on {self.gpuid}")
             instance_segmentation(*self.inputs)
         elif self.method == 'segmentation_validation':
-            segmentation_validation_michael(self.inputs, self.gpuid, 'unfiltered')
+            segmentation_validation_michael(*self.inputs)
 
 
-def main(arguments_):
+def main(method_, raw_dir_, supp_dir_, val_dir_, config_):
+    method = method_
```

Comment:
I usually do this to avoid confusion over the higher scope and to avoid "shadowing".  Pycharm also complains about shadowing if we remove the underscores (maybe it's PEP8?).